src/settings/_box-style.scss

```diff
@@ -4,8 +4,10 @@
 $bdrs-none: 0 !default;
 $bdrs-rounded: 50% !default;
 
-$bdrs-m: 2px !default;
 $bdrs-l: 4px !default;
+$bdrs-m: 2px !default;
+$bdrs-s: 1px !default;
+$bdrs-xs: $bdrs-none !default;
```

Comment:
the right values are here https://github.com/SUI-Components/UX-Definitions/blob/master/Tokens/Border-radius.md#types

Reply:
I know but change `$bdrs-l` from 4px to 50% will be a breaking change. This is basically the same but now we can override the vars in the MA theme. without create a Major of SUITheme.